src/Tween.ts

```diff
@@ -348,9 +347,28 @@ export class Tween<T extends UnknownProps> {
 			this._onStartCallbackFired = true
 		}
 
-		elapsed = (time - this._startTime) / this._duration
-		elapsed = this._duration === 0 || elapsed > 1 ? 1 : elapsed
+		const elapsedTime = time - this._startTime
+		const durationAndDelay = this._duration + (this._repeatDelayTime ?? this._delayTime)
+		const totalTime = this._duration + this._repeat * durationAndDelay
 
+		const calculateElapsedPortion = () => {
+			if (this._duration === 0) return 1
+			if (elapsedTime > totalTime) {
+				return 1
+			}
+
+			const timesRepeated = Math.trunc(elapsedTime / durationAndDelay)
+			const timeIntoCurrentRepeat = elapsedTime - timesRepeated * durationAndDelay
```

Comment:
@MasatoMakino Hello! I renamed some vars for easier reading. F.e. I renamed `elapsedTime` to `timeIntoCurrentRepeat`.

Reply:
I agree with what @trusktr said.

Reply:
I feel like the naming isn't quite right yet. 🤔